packages/compass-components/src/components/resizeable-sidebar.tsx

```diff
@@ -1,10 +1,7 @@
 import React, { useCallback, useState } from 'react';
-import {
-  ResizeHandle,
-  ResizeDirection,
-  uiColors,
-  css,
-} from '@mongodb-js/compass-components';
+import { ResizeHandle, ResizeDirection, css } from '../index';
+
+import { uiColors } from '@leafygreen-ui/palette';
```

Comment:
I got the weirdest errors when trying to import this from ../index along with everything else where it gets re-exported. That suddenly went away when I imported straight from leafygreen-ui/palette..